src/connector/src/sink/google_pubsub.rs

```diff
@@ -273,12 +292,10 @@ impl FormattedSink for GooglePubSubPayloadWriter {
                     ..Default::default()
                 };
                 let awaiter = self.publisher.publish(msg).await;
-                awaiter
-                    .get()
-                    .await
-                    .context("Google Pub/Sub sink error")
-                    .map_err(SinkError::GooglePubSub)
-                    .map(|_| ())
+                self.add_future
```

Comment:
google-pub-sub provides method [publish_bulk](https://docs.rs/google-cloud-pubsub/latest/google_cloud_pubsub/publisher/struct.Publisher.html#method.publish_bulk). Maybe we can call it in batch.